### PR TITLE
2.8.0 notes added

### PR DESCRIPTION
**Pull request by tomjebo**
assuming I'll tag tomorrow (28th)

### CODE REVIEW

**Comment by tomjebo** [comment]
@twsouthwick can you please take a quick look?

**Comment by twsouthwick** [comment]
Can you update the date in the README.md as well?

**Comment by tomjebo** [comment]
@twsouthwick sorry forgot that. done. please review. :)

File: DocumentFormat.OpenXml.Package.props [hunk 1]
```diff
@@ -16,9 +16,14 @@
     <Copyright>Copyright 2017 (c) Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>openxml office</PackageTags>
     <PackageReleaseNotes><![CDATA[
-Version 2.7.3 : *In development*
-- Manual saving was fixed when autosave is false
+
```
Comment (twsouthwick) [inline DocumentFormat.OpenXml.Package.props]: nit: extra newline

**Comment by twsouthwick** [comment]
When you merge, make sure the notes make sense (it defaults to the comment names, which aren't usually needed)

File: DocumentFormat.OpenXml.Package.props (hunk 1 above)
Comment (tomjebo) [reply DocumentFormat.OpenXml.Package.props]: oops. thanks.

**Pull request merged**